dev/com.ibm.ws.microprofile.config.1.2_fat_tck/publish/servers/FATServer/server.xml

```diff
@@ -0,0 +1,24 @@
+<!--Copyright (c) 2017 IBM Corporation and others. All rights reserved.
+    This program and the accompanying materials are made available under the 
+    terms of the Eclipse Public License v1.0 which accompanies this distribution, 
+    and is available at 
+        http://www.eclipse.org/legal/epl-v10.html 
+    Contributors: 
+        IBM Corporation - initial API and implementation
+-->
+<server>
+    <featureManager>
+        <feature>servlet-3.1</feature>
+        <feature>componenttest-1.0</feature>
+        <feature>webProfile-7.0</feature>
+        <feature>localConnector-1.0</feature>
+        <feature>cdi-1.2</feature>
+        <feature>mpConfig-1.2</feature>
+    </featureManager>
+
+    <include location="../fatTestPorts.xml" />
+
+    <!--Java2 security-->
+    <javaPermission className="java.security.AllPermission"  name="*" actions="*" />
```

Comment:
This is needed for Arquillian getClassloader to be allowed in a Java 2 Security build.